### PR TITLE
Greeting languages [WIP]

### LINKED ISSUE #319: "hello" translations & rotations
## Issue - Enhancement
- [x] rotate languages
- [x] create hover feature that shows (in English) what language the greeting is in

### PR DESCRIPTION
**Pull request by ackernaut**
Once merged, this fixes #319:

- [x] Updates team data (greeting string is now an object)
- [x] Adds dotted styling to the greeting content
- [ ] ~Adds tooltip with translation message~

### CODE REVIEW

**Comment by ackernaut** [comment]
@awmichel I’m thinking we need 2 things: a migration to update existing teams (each greeting is now an object with two properties, `content` and `language`) and update the scheme/mutation?

**Comment by ackernaut** [comment]
Ha, false alarm @awmichel (oh, autocomplete!) that was meant for you @mattkrick …

but 👋 @awmichel hope all is well. :)

File: src/universal/utils/makeCheckinGreeting.js [hunk 1]
```diff
@@ -27,7 +72,8 @@ const questions = [
   'What are you cooking next',
   'Learn any new slang lately',
```
Comment (jordanh) [inline src/universal/utils/makeCheckinGreeting.js]: We've got good data the the slang question is hard for folks/too young. I think we should nuke it.

**Comment by jordanh** [comment]
How can I help?

**Comment by jordanh** [comment]
I just rebased this branch to `master`

**Comment by ackernaut** [comment]
ok @jordanh thanks for the help.

Currently, when you hover you see a ❓ cursor and get a native browser tooltip:

GIF

I think this is ready to merge. I’m punting on the Tooltip component. Going to scrub or write an issue where we have a Tooltip module much like the Menu module that is based on portals. Then an issue to go back and at that at the right places (including here). I don’t think it makes sense to focus on that now in terms of priority but maybe I’ll come around to it.

**Comment by ackernaut** [comment]
Ah looks like I may need to update a test…looking now.

**Comment by ackernaut** [comment]
Not sure how to proceed on failing tests here.

**Pull request merged**